docs/ums/README.md

```diff
@@ -125,15 +183,28 @@ export UMS_TOKEN=`curl -X POST -d '{
 ## Fetch a User
 To fetch an user you can look at the following methods:
 
-<CodeSwitcher :languages="{js:'JavaScript',bash:'cURL'}">
+<CodeSwitcher :languages="{js:'JavaScript',py:'Python',bash:'cURL'}">
 <template v-slot:js>
 
 ``` js
 // via alias
-const user = await ums.getUser('Elon Musk'); 
+ums.getUser('Elon Musk').subscribe(user => {}, error=>{});    
 // via email 
-const user = await ums.getUser('elon@tesla.com'); 
+ums.getUser('elon@tesla.com').subscribe(user => {}, error=>{});   
 ```
+</template>
+<template v-slot:py>
+
+``` py
+...
```

Comment:
I would suggest to use full example of code everytime, because some users can open this part of documentation by directly link. And in this case they need to find another part of code.

Reply:
next version